data/i18n/en/en.toml

```diff
@@ -36,6 +36,9 @@ other = " documentation is no longer actively maintained. The version you are cu
 [deprecation_file_warning]
 other = "Deprecated"
 
+[dockershim_message]
+other = """The dockershim code required to run Docker Engine has been removed from the Kubernetes project as of release 1.24. Read the <a href="/dockershim">Dockershim Removal FAQ</a> for further details."""
```

Comment:
```suggestion
other = """The dockershim code required for Kubernetes to use Docker Engine has been removed from the Kubernetes project as of release 1.24. Read the <a href="/dockershim">Dockershim Removal FAQ</a> for further details."""
```

Reply:
@reylejano I like this. I think we can refine it somewhat for clarity and let the FAQ do the heavy lifting.
```suggestion
other = """Dockershim has been removed from the Kubernetes project as of release 1.24. Read the <a href="/dockershim">Dockershim Removal FAQ</a> for further details."""
```

Reply:
I prefer @zacharysarah 's suggestion over mine :)

Reply:
Okay. I used @zacharysarah wording and did a new commit. Thanks!